packages/ckeditor5-table/src/tablelayout/commands/tabletypecommand.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module table/tablelayout/commands/tabletypecommand
+ */
+
+import { Command } from 'ckeditor5/src/core.js';
+
+import { getSelectionAffectedTable } from '../../utils/common.js';
+
+export type TableType = 'layout' | 'content';
```

Comment:
Let's import this type from :point\_right: [https://github.com/ckeditor/ckeditor5/pull/18121#discussion_r1998622816](https://github.com/ckeditor/ckeditor5/pull/18121#discussion_r1998622816) (when [this PR](https://github.com/ckeditor/ckeditor5/pull/18121) will merged).

Reply:
Done in 130e16a.